premake5.lua

```diff
@@ -88,6 +90,11 @@ filter({"configurations:Release", "platforms:Windows"})
   linkoptions({
     "/NODEFAULTLIB:MSVCRTD",
   })
+  buildoptions({
+	"/Ot",
```

Comment:
Tab instead of spaces.

Reply:
Are you saying that you want tabs instead of spaces?

Reply:
No, I'm saying there is a tab instead of spaces. The rest of the file is space delimited.